extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js

```diff
@@ -0,0 +1,165 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.veles', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.wrappers.prebid',
+	'ext.wikia.adEngine.video.vastUrlBuilder',
+	'wikia.geo',
+	'wikia.instantGlobals',
+	'wikia.window'
+], function (adContext, prebid, vastUrlBuilder, geo, instantGlobals, win) {
+	'use strict';
+
+	var bidderName = 'veles',
+		slots = {
+			// Order of slots is important - first slot name in group will be used to create ad unit
+			oasis: {
+				INCONTENT_PLAYER: {
+					sizes: [
+						[ 640, 480 ]
+					]
+				},
+				INCONTENT_LEADERBOARD: {
+					sizes: [
+						[ 640, 480 ]
+					]
+				}
+			},
+			mercury: {
+				MOBILE_IN_CONTENT: {
+					sizes: [
+						[ 640, 480 ]
+					]
+				}
+			}
+		};
+
+	function parseParameters(adParameters) {
+		var pairs = [],
+			parameters = {};
+
+		if (adParameters.childNodes.length) {
+			pairs = adParameters.childNodes[0].nodeValue.split(',');
+		}
+		pairs.forEach(function (pair) {
+			var data = pair.split('=');
+
+			parameters[data[0]] = data[1];
+		});
+
+		return parameters;
```

Comment:
parsing query string looks like more generic functionality, what about moving it to some utils?